src/App.elm

```diff
@@ -68,11 +68,17 @@ update msg model =
         FetchMeLoaded result ->
             case result of
                 Ok me ->
-                    ( { model
-                        | loggedIn = LIUpdate.setMe model.loggedIn me
-                      }
-                    , Cmd.none
-                    )
+                    case model.authStatus of
+                        LoggedIn loginInfo ->
+                            ( { model
+                                | loggedIn = LIUpdate.setMe model.loggedIn me
+                              }
+                            , fetchConversations loginInfo me.id
+                                |> Task.perform FetchConversations FetchConversations
+                            )
+
+                        _ ->
```

Comment:
Please ignore the examples I have set - this is a good spot to avoid `_ ->` and just use `LoggedOut ->` since the compiler will warn us if we neglect something whereas `_ ->` is kind of a wildcard that lets things slip through